tests/logging_test.py

```diff
@@ -48,6 +48,24 @@ def jax_debug_log_modules(value):
   finally:
     jax.config.update("jax_debug_log_modules", original_value)
 
+@contextlib.contextmanager
+def jax_logging_level(value):
+  # jax_logging_level doesn't have a context manager, because it's
+  # not thread-safe. But since tests are always single-threaded, we
+  # can define one here.
+  original_value = jax.config.jax_logging_level
+  original_logging_value = logging.getLogger("jax").level
+  jax.config.update("jax_logging_level", value)
+  try:
+    yield
+  finally:
+    # in case original_value is None, which skips setting logging value
+    # we also set the logging value directly pulled from logger.level
+    jax.config.update("jax_logging_level", str(original_logging_value))
```

Comment:
Wait why doesn't None do anything? Shouldn't that reset to default?

Reply:
Ok, I think I fixed this by using "NOTSET" instead for the default config value